consensus/integration/epoch_qc_test.go

```diff
@@ -0,0 +1,235 @@
+package integration_test

```

Comment:
This isn't related to `consensus` really. I would suggest moving this to `module/epochs`